low-level/solution/lr2/server/health.py

```diff
@@ -602,115 +621,28 @@ def get_nw_status(self, nw_interface, interface):
 
     def get_cortx_service_info(self):
         """Get cortx service info in required format."""
-        service_info = []
-        cortx_services = Service().get_cortx_service_list()
-        for service in cortx_services:
-            response = self.get_systemd_service_info(service)
-            if response is not None:
-                service_info.append(response)
-        return service_info
+        cortx_services = self.service.get_cortx_service_list()
+        cortx_service_info = self.get_service_into(cortx_services)
```

Comment:
This is again changed to `get_service_into`, please correct it to `get_service_info`.

Reply:
Done